modules/seen.py

```diff
@@ -19,7 +19,7 @@ def f_seen(jenni, input):
 
     if not input.group(2):
         return jenni.say('Please provide a nick.')
-    nick = input.group(2).lower()
+    nick = input.group(2).lower().split(' ')[0]
```

Comment:
It seems like we could just change the regex to split on \w (word boundary) and group two could be correct